model/Versioned.php

```diff
@@ -796,8 +796,11 @@ public function canViewVersioned($member = null) {
 		}
 
 		// If we weren't definitely loaded from live, and we can't view non-live content, we need to
-		// check to make sure this version is the live version and so can be viewed
-		$latestVersion = Versioned::get_versionnumber_by_stage($this->owner->class, 'Live', $this->owner->ID);
+		// check to make sure this version is the live version and so can be viewed.
+
+		// consider multiple- and single-stage versioning
+		$versionStage = count($this->stages) > 1 ? $this->liveStage : 'Stage';
```

Comment:
I think if count($this->stages) < 2 then we don't even need to check the version. We probably just `return true` at that point. :D